pkg/infra/tower.go

```diff
@@ -16,9 +16,8 @@ type Tower struct {
 	maxPlatFood    float64
 	currPlatFloor  int
 	agentCount     int
-	agents         map[string]*Base
+	Agents         map[string]Agent
```

Comment:
Why make this public? think its better to have it private.

Reply:
Can't access it in simEnv unless it's exported.

Reply:
Also - custom agents can't access it anyway